homeassistant/components/media_player/sonos.py

```diff
@@ -72,8 +73,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     add_devices(devices)
     _LOGGER.info('Added %s Sonos speakers', len(players))
 
-    def group_players_service(service):
-        """Group media players, use player as coordinator."""
+    def _apply_service(service, service_func, *service_func_args):
```

Comment:
I'd like a better way altogether for handling and registering services, it's not very clear which services are available without digging deep into the implementation in some cases.